src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -2602,18 +2602,10 @@ namespace NServiceBus.Timeout.Core
 }
 namespace NServiceBus.TransportDispatch
 {
-    public class static HeaderContextExtensions
```

Comment:
If I remember correctly we have them in the Upgrade Guide and Doco. So we need to update that